gatsby-node.js

```diff
@@ -37,43 +37,6 @@ exports.onCreateNode = ({ node, actions, getNode }) => {
   postNodes.push(node);
 };
 
-exports.setFieldsOnGraphQLNodeType = ({ type, actions }) => {
-  const { name } = type;
-  const { createNodeField } = actions;
-  if (name !== 'MarkdownRemark') {
-    return;
-  }
-
-  const sortedNodes = postNodes.sort(compareByDate);
```

Comment:
Funkce `compareByDate` není použitá. Smím poprosit o smazání importu a její implementace v `src/helpers/node.js`? Stejně tak se dá odstranit nepoužitý `getNode` v `onCreateNode`.